main.py

```diff
@@ -0,0 +1,21 @@
+
+N = 0
+while int(N) not in range(2, 7):
+    N = input("Enter a number between 2 to 6: ")
+deck = set()
+scores = list()
+for i in range(40):
+    deck.add(str(i+1))
+for i in range(int(N)):
+    card_text = deck.pop()
+    card_number = int(card_text)
+    scores.append(card_number)
+    curr_player_index = str(i+1)
```

Comment:
In large projects, shortcuts for names are might be very confusing. it cleaner not to use the.
 curr -> current.

Reply:
In python we dont declare on the type of the variable, we should understand it from the naming. 
I can understand from the name that "curr_player_index" is a number, but the code show be it is a string. try rename the variable or change the code.